addons/website/static/src/client_actions/website_preview/new_content_systray_item.js

```diff
@@ -154,18 +154,28 @@ export class NewContentSystrayItem extends Component {
         })());
 
         proms.push((async () => {
+            if (!this.website.isRestrictedEditor) {
+                return;
+            }
```

Comment:
In fact it was not the best solution to return here, because this code here is used to set the elements `isDisplayed` property to true or false, so if we just return, we never set it to false to hide them and they are displayed :grimacing: 
So,
- either move this higher at the beginning of toggleDropdown, to prevent opening the dropdown at all (you can simply return or throw an error, as you prefer)
- or set all the access to false if we are restricted, like
  ``` js
  if (isrestrictedEditor) {
     modelsTocheck.map(model => [model, false])
  } else {
      // your user check
  }
  ```
or in the map itself or in any other way that would set the `isDisplayed` to `false` :wink: